src/winforms/toga_winforms/widgets/numberinput.py

```diff
@@ -0,0 +1,31 @@
+from ..libs import WinForms, ClrDecimal
+
+from toga.interface import NumberInput as NumberInputInterface
+
+from .base import WidgetMixin
+
+
+class NumberInput(WidgetMixin, NumberInputInterface):
+    _IMPL_CLASS = WinForms.NumericUpDown
+
+    def __init__(self, id=None, style=None, min_value=0, max_value=100, step=1,
+                 **ex):
+        super().__init__(id=id, style=style, min_value=min_value,
+                         max_value=max_value, step=step, **ex)
+        self._create()
+
+    def create(self):
+        self._impl = self._IMPL_CLASS()
+        self._impl._interface = self
+        self.rehint()
+
+    def _get_value(self):
+        return self._impl.Value
```

Comment:
Does this *return* an integer? Or a ClrDecimal? Is there a need to reverse the cast?